### DISCUSSION TITLE
Render size of WebGL Canvas

### DISCUSSION THREAD

**Discussion by KevinYuki** [jeffreylanters/react-unity-webgl#518] [Help Wanted!]
Hey! I need some help rendering the WebGL canvas to take all screen space available. So I set the width and height of the canvas to 100%. The width works perfectly, but the height has a strange behaviour: it keeps growing like in the video below.

My Unity component:
```typescript
     <Unity
        style={{
          visibility: isLoaded ? "visible" : "hidden",
          height: "100%",
          width: "100%",
        }}
        unityProvider={unityProvider}
      />
```

Does anybody have any suggestions about how I can make the canvas take all the available space?

https://github.com/jeffreylanters/react-unity-webgl/assets/36849090/23e32dbd-0448-411a-92a7-2e4e3ccd0727

**Comment by jeffreylanters** [comment]
Hi. This is a know issue on Unity's side. Try wrapping it into a div with the desired size as following:

```jsx
<div style={{ width: "100%", height: "100%" }}>
  <Unity style={{ width: "100%", height: "100%" }} />
</div>
```
